Bootstrap v4 - (Cards) Card - Body.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "Card - Body",
+    "heading": "card-body",
+    "description": "A card is a flexible and extensible content container. It includes options for headers and footers, a wide variety of content, contextual background colors, and powerful display options.\r\n\r\nIf you\u2019re familiar with Bootstrap 3, cards replace our old panels, wells, and thumbnails. Similar functionality to those components is available as modifier classes for cards.",
+    "usage": "Cards are built with as little markup and styles as possible, but still manage to deliver a ton of control and customization. Built with flexbox, they offer easy alignment and mix well with other Bootstrap components.\r\n\r\nCards have no fixed width to start, so they\u2019ll naturally fill the full width of its parent element. This is easily customized with our various sizing options.\r\n\r\n",
+    "snippet": "\"<div class=\"\"card\"\">\r\n  <div class=\"\"card-body\"\">\r\n    This is some text within a card block.\r\n  <\/div>\r\n<\/div>\r\n\"\r\n",
```

Comment:
is it normal to have two \"\" in html? that looks wrong.

Reply:
I'm looking at the snippets. I copied from examples, there must have been some where there were open and closing ""

Reply:
Looking at the url where the details come from it does not have the double quotes so yea somewhere there must have been an issue. Listen I can fix this very quickly, instead of doing it one by one, I can do it with bash... will save you lots of time.

Reply:
I'm a 1/3 of the way through on editing them already. As far as the URL, this was a direct export, using Share Snippets, with no editing.